lib/rpmsg/rpmsg_internal.h

```diff
@@ -35,6 +35,9 @@ extern "C" {
 	} while (0)
 #endif
 
+#define RPMSG_BUF_HELD (1U << 31) /* Flag to suggest to hold the buffer */
```

Comment:
The resolved field is used in two phase:
1.

> This PR need few reworks but LGTM after that.
> 
> FYI some additional updates are needed related to coding rules put in place since last release.
> Please have a look to the [checkpatch result](https://github.com/OpenAMP/open-amp/pull/162/checks?check_run_id=619426784)
> No error but some warnings that should be fixed to respect coding rules we just put in place ( I need to find a way to a status failed on warnings)
> gitlint --commits e2ebdda75394b06004a7377dac70e5d0af49f823-HEAD also reports 2 issues on you commit messages.
> 
> Thanks!

Sure, I will provide the update to fix the style issue.